src/components/users/UserCreateForm.vue

```diff
@@ -60,17 +63,45 @@ const user = ref<CCFUserCreate>({} as CCFUserCreate);
 
 const emit = defineEmits<{
   cancel: [];
-  create: [user: CCFUserCreate];
+  create: [user: CCFUser];
+  error: [error: string];
 }>();
 
+const instance = useApi();
+const { execute } = useAxios<DataResponse<CCFUser>>("/api/users",
+  {
+    method: "POST",
+    headers: {'Content-Type': 'application/json'}
+  },
+  instance,
+  {
+    immediate: false
+  }
+);
 
-function checkUser() {
+async function createUser() {
   if (passwords.error) {
     return;
   }
-
   user.value.password = passwords.password;
-  emit('create', user.value);
+
+  try {
+    const response = await execute({
+      data: user.value,
+    });
+
+    if (!response.data.value) {
+      emit('error', "Failed to create user - no user was returned");
+      return;
+    }
+
+    emit('create', response.data.value?.data)
+  } catch (error) {
+    const errorResponse = error as AxiosError<ErrorResponse<ErrorBody>>;
+    console.error('Error creating user:', errorResponse.response?.data.errors.body);
+    emit('error', errorResponse.response?.data.errors.body ?? 'Unknown error occurred');
```

Comment:
Error shouldn't need to be propagated to the parent component, unless there is a good reason for it. In this case the create form itself will show the error, and the parent shouldn't really be concerned with it happening at all.